app/views/templates/search/channels_landing.rb

```diff
@@ -56,486 +34,482 @@ def navigation
+                count: "12",
+                media_type: "letters"
               },
               {
-                :title =>"Papageno costumes",
-                :url => root_url + "channels/music?q=Papageno+costumes",
-                :count => "42",
-                :media_type =>"images"
+                title: "Papageno costumes",
+                url: root_url + "channels/music?q=Papageno+costumes",
+                count: "42",
+                media_type: "images"
               },
               {
-                :title =>"Baroque wedding music",
-                :url => root_url + "channels/music?q=Baroque+wedding",
-                :count => "5",
-                :media_type =>"audio recordings"
+                title: "Baroque wedding music",
+                url: root_url + "channels/music?q=Baroque+wedding",
+                count: "5",
+                media_type: "audio recordings"
               }
             ]
           },
-          :promoted =>{
-            :title => "Promoted title",
-            :items =>[
+          promoted: {
+            title: "Promoted title",
+            items: [
               {
-                :title =>"The legacy of Punk",
-                :url => "urlhere",
-                :is_exhibition => true,
-                :bg_image =>"sample/thumb-music.jpg"
+                title: "The legacy of Punk",
+                url: "urlhere",
+                is_exhibition: true,
+                bg_image: "sample/thumb-music.jpg"
               },
               {
-                :title =>"Russian conductors in the EU",
-                :url => "urlhere",
-                :is_exhibition => true,
-                :bg_image =>"sample/thumb-music.jpg"
+                title: "Russian conductors in the EU",
+                url: "urlhere",
+                is_exhibition: true,
+                bg_image: "sample/thumb-music.jpg"
               },
               {
-                :title =>"Famous Concert halls",
-                :url => "urlhere",
-                :is_exhibition => true,
-                :bg_image =>"sample/thumb-music.jpg"
+                title: "Famous Concert halls",
+                url: "urlhere",
+                is_exhibition: true,
+                bg_image: "sample/thumb-music.jpg"
               },
               {
-                :title =>"Pop Music in 1980's Berlin",
-                :url => "urlhere",
-                :is_exhibition => true,
-                :bg_image =>"sample/thumb-music.jpg"
+                title: "Pop Music in 1980's Berlin",
+                url: "urlhere",
+                is_exhibition: true,
+                bg_image: "sample/thumb-music.jpg"
               },
               {
-                :title =>"Music piracy",
-                :url => "urlhere",
-                :is_exhibition => true,
-                :bg_image =>"sample/thumb-music.jpg"
+                title: "Music piracy",
+                url: "urlhere",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.